example/lib/routes/home_page.dart

```diff
@@ -244,7 +245,12 @@ class _HomePageState extends State<HomePage> with WidgetsBindingObserver {
     const String screenSet = 'Default-RegistrationLogin1';
 
     try {
-      screenSetSubscription = widget.sdk.showScreenSet(screenSet).listen(
+      screenSetSubscription = widget.sdk
```

Comment:
The formatting is a bit off, due to the last comma?

```suggestion
      screenSetSubscription = widget.sdk.showScreenSet(screenSet, isDebug: kDebugMode)
```

Reply:
Fixed this as well